docs/requirements.md

```diff
@@ -0,0 +1,10 @@
+# Requirements
+
+* docker, version >= 1.17 recommended
+* docker-compose >= 1.13.0
+* at least 4 GiB of RAM
+* 2 CPUs (cores) or more recommended
+
+Following presents basic requirements for the **metric pipeline**.
```

Comment:
**The** following presents **the** basic

Reply:
Done